### ISSUE TITLE
'intersect' calls with 'stream' option

### ISSUE THREAD

**Issue by uludag** [daler/pybedtools#195]
_intersect_ calls with _stream_ option does not return the same object when the call made without the _stream_ option:

```python
a = BedTool(example_filename('a.bed'))
b = BedTool(example_filename('b.bed'))

c = a.intersect(b, stream=True)
print(c.count())  # returns 3
c.saveas('c.bed') # saved file is empty
print(c.count())  # returns 0
```
If _stream_ option not used file _c_ above is not empty and has 3 elements.

**Comment by daler** [comment]
`stream=True` returns an iterator that is consumed when the `count` method is used. See https://daler.github.io/pybedtools/topical-iterators.html, which actually uses this exact example to describe what's happening (though it uses `len` rather than `.count()`)

**Comment by uludag** [comment]
Thanks for your reply. Although I have earlier seen the page you pointed I did not read it carefully; I was unable to think that the behavior I saw was an intended feature of the library.

I also have noticed an unrelated minor issue that I do not want to open a new formal issue; when I browse some of the documentation pages the section called 'Original BEDTools help' is displayed in a box that not sized well, which makes reading difficult,  e.g. https://daler.github.io/pybedtools/autodocs/pybedtools.bedtool.BedTool.intersect.html
